tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -511,6 +511,7 @@
                                 "16",
                                 "17",
                                 "18",
+                                "19",
```

Comment:
@elipe17 why this change? 

![WPS_49](https://github.com/raft-tech/TANF-app/assets/63075587/b497513f-e61c-416a-8283-30d2e865fd44)